plugins/libvirt/libvirt_test.go

```diff
@@ -0,0 +1,24 @@
+package libvirt
+
+import (
+	"testing"
+
+	"github.com/influxdb/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLibvirt(t *testing.T) {
```

Comment:
Does this test have dependencies? Does the host running the tests need to have libvirt installed?